design/comments.go

```diff
@@ -60,16 +60,10 @@ var createCommentAttributes = a.Type("CreateCommentAttributes", func() {
 })
 
 var commentRelationships = a.Type("CommentRelations", func() {
-	a.Attribute("created-by", commentCreatedBy, "This defines the created by relation")
+	a.Attribute("creator", relationGeneric, "This defines the creator of the comment")
```

Comment:
@DhritiShikhar you know what? Now that you pointed me to [this snippet](https://github.com/fabric8-services/fabric8-wit/blob/ba2ef35742a581efdac21170a06d05d3fed111e4/design/workitems.go#L34) in the work item design, I think you might as well go ahead with `relationGeneric` ;)

```go
// WorkItemRelationships defines only `assignee` as of now. To be updated
var workItemRelationships = a.Type("WorkItemRelationships", func() {
	a.Attribute("assignees", relationGenericList, "This defines assignees of the Work Item")
	a.Attribute("creator", relationGeneric, "This defines creator of the Work Item")
	a.Attribute("baseType", relationBaseType, "This defines type of Work Item")
	a.Attribute("comments", relationGeneric, "This defines comments on the Work Item")
	a.Attribute("iteration", relationGeneric, "This defines the iteration this work item belong to")
	a.Attribute("area", relationGeneric, "This defines the area this work item belongs to")
	a.Attribute("children", relationGeneric, "This defines the children of this work item")
	a.Attribute("space", relationSpaces, "This defines the owning space of this work item.")
})
```

Sorry for having been so picky.